### ISSUE TITLE
Copy of 10-7959a: Health insurance information

### ISSUE THREAD

**Issue by marywang2** [department-of-veterans-affairs/va.gov-team#86956]
As a VA.gov user, I want to fill out the CHAMPVA claim form with the sponsor information so that I can file a claim.

[Figma link](https://www.figma.com/design/Tfhq5h2LwXEeEEtFBAAFOv/CHAMPVA-Claims-(10-7959a)?node-id=961-139667&t=Ko13IszQmJiM9ZRY-0)

- Page elements:
  - breadcrumb 
  - header and subhead
  - copy
  - footer
- Sponsor information
  - name/dob

Acceptance criteria:

- Given the user is on VA.gov and about to fill out the CHAMPVA claim form
- When the user gets to the sponsor info page
- Then the user can fill out the sponsor's information

**Comment by michaelclement** [comment]
Work is done in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30718). Waiting on:
- [ ] Team approval
- [ ] Platform approval
 
@syd-hoeper @marywang2 There are some notable issues to discuss:
1. The design system v3 file upload (the drag & drop one) only supports a single file upload at this time. I've gone ahead and replaced that file upload with the v1 version (identical to 10-10d and our other forms) as it supports multiple file uploads, but I wanted to bring it to your attention.
2. There are some slight style differences between the new list loop and the mockup, specifically the font weight/typeface of the header above the "do you have another?" question, and the non-removable yellow warning banner stating the user has met the max number of insurers after adding 2 (see screenshot below). 

<img width="700" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBNU8rQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--4770e62d212da0d45d7e7b60c2bea74a80948791/Screenshot%202024-07-08%20at%2010.08.46.png" alt="Screenshot 2024-07-08 at 10.08.46.png" />

**Comment by syd-hoeper** [comment]
All those changes sound good to me!

For transparency: Michael and I discussed the List and Loop changes in Figma last week and everything seems good to go. Noting that the Figma will be out-of-date and I can update that once we get to the UAT stage.

I'm having trouble getting on staging right now, so I'm not able to see the "Do you have another?" typography issue

**Comment by michaelclement** [comment]
Thanks! Typography issue:
<img width="594" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeW0vQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--0cb5f9bb98ea61d8a36f8166edb139357b1960ff/Screenshot%202024-07-08%20at%2012.05.28.png" alt="Screenshot 2024-07-08 at 12.05.28.png" />

**Comment by michaelclement** [comment]
Ok I'm going to update the file upload to be V3 and will follow up here after - thanks!

**Comment by michaelclement** [comment]
After looking into it more, seems like this multi-file component hasn't really been used in other form-library forms yet, so I'll have to spend a bit of extra time getting it integrated. I'll keep updates here as this progresses, but it'll extend the time needed for this ticket a bit

**Comment by michaelclement** [comment]
upped the points because of file upload stuff - still digging into the best way to implement the new version

**Comment by michaelclement** [comment]
At this point, there appears to be a bug in the v3 upload component that will have to be resolved by platform (waiting to hear back on Monday for confirmation).

The bug prevents files from actually being deleted after they've been uploaded, which would prevent our actual adoption of this component until it's been fixed up stream. 

@syd-hoeper @marywang2 for awareness - If the above is confirmed, I think we should split the specific implementation of the v3 file component into a separate ticket, put it on the backlog, and proceed with the v1 file component for the time being/evaluate this work. Would that be acceptable?

**Comment by michaelclement** [comment]
Per the above comment, I've created [this ticket](https://app.zenhub.com/workspaces/ivc-forms-652da2d3f0ae4c0016bfb109/issues/gh/department-of-veterans-affairs/va.gov-team/88289) for the move to the v3 file upload component (when the bug is fixed). I'm going to move this ticket to review now since everything is in place with the v1 file upload.

**Comment by michaelclement** [comment]
@marywang2  @syd-hoeper This should be visible on staging at this point and is ready for review. Thanks!

**Comment by marywang2** [comment]
Changes:
- pharmacy claim page (upload) - add below sentence to the 'note' section (with space in between existing copy), remove the 3rd bullet from how to upload files, update additional notes section (replace first bullet with the 2 bullets from figma)
<img width="433" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBeGpkQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--07af4f3f43e6387927117db86622c238d6f88daa/Screenshot%202024-07-15%20at%2010.36.12%E2%80%AFAM.png" alt="Screenshot 2024-07-15 at 10.36.12 AM.png" />

**Comment by syd-hoeper** [comment]
Yeah I'm also not seeing the medical upload page, just the EOB pages. Agree with everything Mary said above. All the upload stuff looks good to me

**Comment by michaelclement** [comment]
Requested copy changes have been implemented in [this PR](https://github.com/department-of-veterans-affairs/vets-website/pull/30839) (see PR for screenshots of changes)

Waiting on
- [ ] Team approval
- [ ] Platform approval

@syd-hoeper @marywang2 re: the page not showing up, I'm able to see it on staging... Not sure if it's possible y'all are not seeing the latest changes somehow? But regardless, maybe after this next PR goes through we can check again and double-verify. Here's what I'm seeing:

<img width="875" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemZlQlE9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--997708a3da07ebb612a702cbb2d278491ddded7f/med_claim.png" alt="med_claim.png" />

**Comment by michaelclement** [comment]
Latest changes have been merged in, are y'all able to see the page yet? @marywang2 @syd-hoeper

**Comment by syd-hoeper** [comment]
I can see it now!

**Comment by marywang2** [comment]
I see it now too! Thanks @michaelclement!
